src/history/git.js

```diff
@@ -45,6 +45,24 @@ export default class Git {
            || (status.not_added.indexOf(this.relativePath(filepath)) > -1);
   }
 
+  async log(options = {}) {
+    try {
+      options.file = options.file && this.relativePath(options.file);
+      const logSummary = await this.git.log(options);
+      return logSummary.all;
+    } catch (error) {
+      if (!error.message.includes('unknown revision or path not in the working tree')) {
+        throw (error);
+      }
+      return [];
+    }
+  }
+
+  async isTracked(filepath) {
+    const result = await this.git.raw('ls-files', this.relativePath(filepath));
```

Comment:
> If the file has been `git add`-ed, without having been committed, it will still be returned by `ls-files`. I hope that's expected 🙂

Reply:
Solution: use `git ls-files --modified`.

Reply:
Do not work for already committed file that are not modified